app/scaffold-www/js/left-drawer.js

```diff
@@ -0,0 +1,129 @@
+                    var item = menuItems[i];
+                    buildHtmlForItem(item);
+                }
+            };
+
+            var buildHtmlForItem = function(item) {
+                pr('<li>');
+                currentTabDepth++;
+                if(item.subItems && item.subItems.length > 0) {
+                    buildNextButton(item.title);
+                    pr('<ul>');
+                    currentTabDepth++;
+
+                    buildHeader(item.title);
+
+                    parseData(item.subItems);
+                    currentTabDepth--;
+                    pr('</ul>');
+                } else {
+                    pr('<a class="navitron-link" href="' + item.href + '">' + item.title + '</a>');
+                }
+                currentTabDepth--;
+                pr('</li>');
+            };
+
+            var buildNextButton= function(title) {
+                pr('<button class="navitron__next-pane" type="button">' + title + '</button>');
+            };
+
+            var buildHeader = function(title) {
+                pr('<li class="navitron__header">');
+                currentTabDepth++;
+                pr('<div class="c-bar">');
+                currentTabDepth++;
+                pr('<button class="c-button navitron__prev-pane" type="button">Back</button>');
+                pr('<span class="c-bar__heading">' + title + '</span>');
+                currentTabDepth--;
+                pr("</div>");
+                currentTabDepth--;
+                pr('</li>');
+            };
+
+            parseData(menuItems);
+            return generatedHtmlString;
+        };
+
+        var setupNavitron = function() {
+            // Setup the glue required to open selected items in the main navigation view
+            $('.navitron-link').on('click', function(event) {
+                var url = event.target.href;
+                var title = event.target.innerText;
+                var navigateToNewRootView = Astro.jsRpcMethod('navigateToNewRootView', ['url', 'title']);
+                navigateToNewRootView(url, title).error(function(error) {
+                    console.log('error', error);
+                });
+                event.preventDefault();
+            });
+
+            // Run the secret sauce to create the navitron
+            $('#leftMenuNavitron').navitron({
+                structure: false,
+                shiftAmount: 100
+            });
+        };
+
+        var renderNavitron = function(menuItems) {
+            var generatedHtml = generateNavitronHtml(menuItems);
+            $('#generatedNavitronHtmlPlaceholder').replaceWith(generatedHtml);
+            setupNavitron();
+        };
+
+        Astro.jsRpcMethod('menuItems', [])().then(function(menuItems) {
+            renderNavitron(menuItems);
+        });
+
+        // Allows the left drawer to be rerendered by having the drawerController
+        // call `renderLeftMenu`.
+        Astro.on('setMenuItems', function(params) {
```

Comment:
@jvoll @lizcross and I discussed and decided it could be done in another ticket since these menus are not being dynamically generated. I will open a ticket for this!